scripts/stage1.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+function die
+{
+	echo "$1" >&2
+	exit 1
+}
+
+[[ $# -eq 1 ]] || die "Illegal number of parameters"
+target="$1"
+workdir=$(mktemp -d)
+img=$workdir/img
+base="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+#
+# Set up working directory
+#
+cd $workdir
+sudo mkdir img
+sudo chmod 0755 img
+
+#
+# Perform installation process
+#
+cd $base
+sudo rsync -a ../base_img/ $img/
+
+sudo mkdir -p $img/bin/
+
+for file in /bin/busybox /bin/kmod /bin/systemd-tmpfiles /bin/udevadm \
+	/lib/systemd/systemd-networkd /lib/systemd/systemd-udevd \
+	/usr/sbin/dropbear /usr/lib/dropbear/dropbearconvert; do
+	sudo mkdir -p $img/$(dirname $file)
+	sudo rsync -a $file $img/$file
+	./get_deps $img/$file $img
+done
+sudo ln -rs $img/bin/busybox $img/bin/sh
+
+sudo rsync -aL /lib64/ld-linux-x86-64.so.2 $img/lib64/
+sudo mkdir -p $img/lib/x86_64-linux-gnu/
+ls $img/lib/x86_64-linux-gnu/
+sudo rsync -aL /lib/x86_64-linux-gnu/libnss* $img/lib/x86_64-linux-gnu/
+ls $img/lib/x86_64-linux-gnu/
+sudo rsync -a /lib/modprobe.d $img/lib/
+
+sudo rsync -a /etc/{group,passwd,shadow}{,-} $img/etc/
+sudo sed -i 's@/bin/bash@/bin/sh@g' $img/etc/passwd{,-}
+sudo sed -i 's@/opt/delphix/server/bin/supportlogin@/bin/sh@g' $img/etc/passwd{,-}
```

Comment:
This puts a dependency on the virtualization package, right?

Additionally, is this script run during the package build phase? If so, this will use the file from the build server, right (I see it called only from the Makefile)? Is that correct? I'm not sure I fully understand where/when this script will run...

Reply:
This code should be in the postinst script, I've also added a comment that clarifies which script is run where.